packages/lab/src/typography/Text.css

```diff
@@ -107,35 +104,42 @@ h4.uitkText small,
 /* Label-Caption */
 label.uitkText,
 .uitkText-label {
+  --text-lineHeight: var(--uitk-text-caption-lineHeight);
   font-size: var(--uitk-text-caption-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-caption-lineHeight));
 }
 
 /* Help text */
 .uitkText-helpText.uitkText {
-  font-style: var(--uitk-text-help-fontStyle);
+  --text-lineHeight: var(--uitk-text-help-lineHeight);
   color: var(--uitk-text-secondary-foreground);
+  font-style: var(--uitk-text-help-fontStyle);
   font-size: var(--uitk-text-help-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-help-lineHeight));
 }
 
 /* Figure 1 */
 .uitkText-figure1.uitkText {
+  --text-lineHeight: var(--uitk-text-figure1-lineHeight);
   font-size: var(--uitk-text-figure1-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-figure1-lineHeight));
   font-weight: var(--uitk-text-figure1-fontWeight);
 }
 
 /* Figure 2 */
 .uitkText-figure2.uitkText {
+  --text-lineHeight: var(--uitk-text-figure2-lineHeight);
   font-size: var(--uitk-text-figure2-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-figure2-lineHeight));
   font-weight: var(--uitk-text-figure2-fontWeight);
 }
 
 /* Figure 3 */
 .uitkText-figure3.uitkText {
+  --text-lineHeight: var(--uitk-text-figure3-lineHeight);
   font-size: var(--uitk-text-figure3-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-figure3-lineHeight));
   font-weight: var(--uitk-text-figure3-fontWeight);
 }
+
+.uitkText {
+  line-height: var(--text-lineHeight);
```

Comment:
shall we give --uitkText-lineHeight here for overrides?

Reply:
I'm not sure about this one. Do we want to let them override line-height? And if so, should we let them override for all text components on all densities?

Reply:
if we define just the token here, then they can set that within the given classnames themselves, we don't need to provide it everywhere as that will bloat it out. but whether we let them do that or not is perhaps a question for design